packages/manager/apps/dedicated/client/app/account/contacts/request/change/user-contacts-request-change.controller.js

```diff
@@ -1,12 +1,12 @@
 import set from 'lodash/set';
 
 angular
-  .module('UserAccount')
-  .controller('UserAccount.controllers.contacts.request', [
+  .module('ovhManagerDedicatedAccount')
+  .controller('ovhManagerDedicatedAccount.controllers.contacts.request', [
```

Comment:
rename ovhManagerDedicatedAccount.controllers.contacts.request to UserAccountsRequestChangeCtrl